packages/grid_client/src/high_level/twinDeploymentHandler.ts

```diff
@@ -34,6 +36,11 @@ class TwinDeploymentHandler {
       }
     }
     try {
+      const kycStatus = await this.kyc.status();
+      if (kycStatus !== KycStatus.verified)
+        throw new ValidationError(
+          "Your account is not verified. Please sign into ThreeFold Dashboard or ThreeFold Connect mobile app to complete your KYC verification.",
+        ); //TODO add kyc manual link
```

Comment:
this is not required, already domains require 2 contracts (name contract to be unique and node contract to deploy it)